modules/http-helper/http_helper.go

```diff
@@ -31,9 +31,18 @@ func HttpGet(t testing.TestingT, url string, tlsConfig *tls.Config) (int, string
 func HttpGetE(t testing.TestingT, url string, tlsConfig *tls.Config) (int, string, error) {
 	logger.Logf(t, "Making an HTTP GET call to URL %s", url)
 
+	// Get a deepcopy of the default Transport implementation
+	DefaultTransporter := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
If `Clone` does a deep copy, couldn't this be simplified to:

```go
tr := http.DefaultTransport.(*http.Transport).Clone()
```

That would make the code more maintainable too in case `http.Transport` adds new fields in the future.

Reply:
Thank you for pointing that out.